packages/calling-component-bindings/src/baseSelectors.ts

```diff
@@ -119,3 +121,12 @@ export const getDiagnostics = (
  */
 export const getCallState = (state: CallClientState, props: CallingBaseSelectorProps): string =>
   state.calls[props.callId]?.state;
+
+/* @conditional-compile-remove(unsupported-browser) */
+/**
+ * @private
+ */
+export const getEnvironmentInfo = (state: CallClientState): EnvironmentInfo | undefined => {
```

Comment:
Question about this baseSelector - is there any component currently using this selector? It makes sense to put it here if there is, if not, then this selector should stay in composite folder

Reply:
Caroline is going to be using it in the errorbar component once this is merged in.

Reply:
So we got a plan to expose these on component layer, nice!